src/engines/nwn2/roster.cpp

```diff
@@ -39,6 +39,42 @@ Roster::Roster() {
 Roster::~Roster() {
 }
 
+/** Return the roster name of the first member. */
+Common::UString Roster::getFirstRosterMember() {
+	Common::UString name = "";
+	_lastRetrieved = 0;
+
+	// Set to the first member's name
+	if (!_members.empty())
+		name = (_members.begin())->rosterName;
```

Comment:
I'd just do a plain `return` of the value there, and a `return ""` for the default case, without going through the temp variable there. You can also use `_members.front().rosterName`, without having to go through the iterator.